stream-chat-android-client/src/main/java/io/getstream/chat/android/client/channel/ChannelClient.kt

```diff
@@ -421,9 +421,12 @@ public class ChannelClient internal constructor(
         return client.queryMembers(channelType, channelId, offset, limit, filter, sort, members)
     }
 
-    public fun getFileAttachments(offset: Int, limit: Int): Call<List<AttachmentWithDate>> =
+    public fun getFileAttachments(offset: Int, limit: Int): Call<List<Attachment>> =
         client.getFileAttachments(channelType, channelId, offset, limit)
 
-    public fun getImageAttachments(offset: Int, limit: Int): Call<List<AttachmentWithDate>> =
+    public fun getImageAttachments(offset: Int, limit: Int): Call<List<Attachment>> =
```

Comment:
These return type changes break public API (though it's very recently public API, so it might not have users yet)

Reply:
`AttachmentWithDate` was created recently and it hasn't been announced and this method is not documented on our webpage yet, I don't think it is an issue. BTW, `AttachmentWithDate` wasn't a good name neither a needed model. It was created because some UI requirements on the sample app